### DISCUSSION TITLE
ReanimatedError: Attempted to assign to readonly property., js engine: reanimated

### DISCUSSION THREAD

**Discussion by s-t-sa** [software-mansion/react-native-reanimated#4888] [Q&A]
Hi,

Had a crash issue with Android using sharedTransitionTag ( cannot find a view tag ) while it worked fine on iOS with version 3.3.0, however taking any version greater than 3.3.0 fixes the Android issue but creates ReanimatedError: Attempted to assign to readonly property., js engine: reanimated for iOS. 3.4.0+ has been out a short while and i don't see it raised as an issue which suggest this version is not taken by many or the issue is not common. 

Anybody seeing this ?
Thanks

**Comment by tungne** [comment]
I'm having an issue, can someone help me fix it?
